test.tex

```diff
@@ -4,7 +4,10 @@
 
 \title{Ceci est un titre}
 \author{Karnaj, Clem}
-\licence{CC-BY-NC-ND}
+\licence[./img/license/cc-by-nc-nd_icon.svg]{CC-BY-NC-ND}{https://creativecommons.org/licenses/by-nc-nd/4.0/legalcode}
```

Comment:
I'm not very found of having the whole set of CC icons packed with the template. One for the test is obviously required, but it would be better to give the responsibility to the person who would like to use the package to provide the icons.

This would avoid two maintenance issues:

+ Infinite set of icons (because one would like the GPL, the LGPL, the ...)
+ Updating the icons when they changes (the CC ones may change in the future)

No ?

Reply:
when the new version of zmarkdown and remark-ping are published, I will update the zmd integration PR which will embed the license icons so do not pack them into latex-template.

Reply:
Yes, but do is pray user use vectorial image instead of bitmap for prevent page layout suck.

Reply:
In fact I use the svg icons, don't worry.
If you want to have One licence to feed the test.tex/test.pdf example feel free to do it, but do not pack all CC licenses.

And add the "svg constraint" to the documentation.

Reply:
I agree with @artragis :)